### ISSUE TITLE
WordPress Multisite Subdomain Problems

### ISSUE THREAD

**Issue by INA2N** [INA2Network/Nginx-Renew-LE-Script#1]
In the WP multisite using subdomains, WP takes issue with the security certificate belonging to the root domain.  I had to create a separate certificate for the FQDN, even though it's part of the multisite environment.

Without this dedicated config, WP won't load properly and throws several errors.  Of course, there has to be a FQDN for the multisite subdomain.

Could this be resolved so only the multisite root domain resolves the FQDN, or is this a limitation of LetsEncrypt?

Doing a little research, it looks as though only one FQDN can be present per certificate submission request/file.

Correct:
domain.com
www.domain.com

Incorrect:
domain.com
www.domain.com
example.com
www.example.com

Can you confirm this?

**Comment by stribika** [comment]
Yes, only one domain name will be in the certificate's "Common Name" field. (Usually the first, unless you already have a valid certificate for the same set of domains with a different Common Name.) The rest will be listed in the "Subject Alternative Name" field. Browsers will still accept the certificate.

**Comment by theibel** [comment]
Thank you for confirming.  I will test the new script for the Nginx config next week.
